doc/examples/gestures/lib/main_draggables.dart

```diff
@@ -42,11 +42,11 @@ class DraggableSquare extends PositionComponent with Draggable {
 
   @override
   bool onDragUpdate(int pointerId, DragUpdateDetails details) {
-    if (!_isDragging) {
+    if (initialPosition != null && !_isDragging) {
       _isDragging = true;
-      dragDeltaPosition = initialPosition - position;
+      dragDeltaPosition = initialPosition! - position;
     }
-    position = details.localPosition.toVector2() - dragDeltaPosition;
+    position = details.localPosition.toVector2() - dragDeltaPosition!;
```

Comment:
I think dragDeltaPosition can be null here can't it?

Reply:
I was unsure, maybe @spydon can elucidate? in general with the gestures examples I wanted to remove booleans that basically act as null checks so that the compiler can be smarter